scripts/jenkins/Jenkinsfile.release

```diff
@@ -7,6 +7,9 @@ properties([
         string(name: 'TAG',
                defaultValue: 'v0.0.0',
                description: 'CLUO tag to release'),
+        booleanParam(name: "PUSH_LATEST",
```

Comment:
Ok, I still don't think that means we're committed to updating a "latest" forever though - we can stop at any point and earlier is better.